plugins/net_plugin/net_plugin.cpp

```diff
@@ -3348,9 +3350,12 @@ namespace eosio {
       }
       g.unlock();
 
-      metrics.num_clients.value = num_clients;
-      metrics.num_peers.value = num_peers;
-      
+      if (net_metrics_listener) {
+         metrics.num_clients.value = num_clients;
+         metrics.num_peers.value = num_peers;
```

Comment:
What about `dropped_trxs`?

Reply:
This is incremented when the trx is dropped - line 2689

Reply:
This should be `post_metrics()` and also call `post_metrics()` inside `if (fc::time_point::now() - fc::seconds(1) >= last_dropped_trx_msg_time)`.